st3/sublime_lib/view_stream.py

```diff
@@ -88,7 +93,7 @@ def read(self, size):
 
     @guard_validity
     @guard_selection
-    def readline(self, size=-1):
+    def readline(self, size: int = -1) -> str:
```

Comment:
I just now noticed that we don't have the `size` parameter documented. Also seems weird to default to `-1` instead of `None`.

Reply:
Definitely should document. `-1` is used in the `io` classes. I would prefer `None`, but I think it's more important to maintain consistency.

Reply:
I agree with that.